app/models/case/sar/offender.rb

```diff
@@ -72,7 +72,8 @@ def searchable_fields_and_ranks
                  number_exempt_pages: :integer,
                  is_partial_case: :boolean,
                  partial_case_letter_sent_dated: :date,
-                 further_actions_required: :string
+                 further_actions_required: :string,
+                 is_rejected: :boolean
```

Comment:
Thoughts are to add this attribute as a boolean. Defines whether a case is rejected.